rdtools/soiling.py

```diff
@@ -337,8 +351,8 @@ def _calc_result_df(self, trim=False, max_relative_slope_error=500.0,
                 'run_slope_high': 0,
                 'max_neg_step': min(run.delta),
                 'start_loss': 1,
-                'inferred_start_loss': run.pi_norm.median(),#changed from mean/Matt
-                'inferred_end_loss': run.pi_norm.median(),#changed from mean/Matt
+                'inferred_start_loss': run.pi_norm.mean(),#changed from mean/Matt
+                'inferred_end_loss': run.pi_norm.mean(),#changed from mean/Matt
```

Comment:
@mdeceglie - This change here from median() to mean() seems to cause the RuntimeWarning: Mean of empty slice